tests/test_llm_seam_isolation.py

```
@@ -0,0 +1,94 @@
+    violations: list[str] = []
+    for py_path in _iter_repo_python_files():
+        rel = py_path.relative_to(REPO_ROOT).as_posix()
+        if rel in _ALLOWED:
+            continue
+        if rel == "ai_service.py":
+            # Temporary exception until drill_chat + generate_repair_reps migrate.
+            continue
+        try:
+            text = py_path.read_text(encoding="utf-8")
+        except Exception:
+            continue
```

Ruff (0.15.12)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)